services/auth/basic.go

```diff
@@ -159,3 +168,19 @@ func validateTOTP(req *http.Request, u *user_model.User) error {
 	}
 	return nil
 }
+
+func GetAccessScope(store DataStore) auth_model.AccessTokenScope {
+	if v, ok := store.GetData()["ApiTokenScope"]; ok {
+		return v.(auth_model.AccessTokenScope)
+	}
+	switch store.GetData()["LoginMethod"] {
+	case OAuth2TokenMethodName:
```

Comment:
How is this case different from below if it just uses fallthrough?

Reply:
I think they are the same, but I leave OAuth2 here because I think in future, OAuth2 token should have their scopes.